source/public/styles/styles2.css

```diff
@@ -0,0 +1,415 @@
+
+/*Styling for Home Page!*/
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+*, *::before, *::after{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0; 
+    font-family: 'Inter', "FontAwesome", sans-serif;
+    
+}
+
+#homepage {
+    height:15%;
+    width:85%; /*Percentage based on parent container size*/
+    margin: 0 auto; /*Sets top and bottom margin to 0. but horizontal auto adjusts according to browser*/
+    display:flex;
+    align-items: center; /*Aligns content vertically in the center of the container*/
+    justify-content: center; /*Aligns flex items horizontally*/
+}
+
+#homepage h1{
+    color: #FF9B83;
+}
+
+main{
+    display:flex;
+    flex-grow: 1;
+    justify-content: space-evenly;
+}
+
+#calendarContainer{
+    display:flex;
+    justify-content: flex-start;
+    border: 1px solid white;
+    width: 50%;
+    border-radius: 10px;
+}
+
+.calendar{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: white;
+    gap: 20px;
+    border-radius: 10px;
+    
+    
+}
+
+.days{
+    display: flex;
+    width: 97%;
```

Comment:
Could you change the width here to `100%`? The days of the week look a bit off-centered from the actual dates and this should fix it